src/containers/SignIn/index.js

```diff
@@ -1,8 +1,110 @@
 import React from 'react';
-import UnderConstruction from '../UnderConstruction';
+import { Link } from 'react-router-dom';
 
-const SignIn = props => {
-  return <UnderConstruction {...props} />;
+import { Form, Input, Icon, Button } from 'antd';
+import PropTypes from 'prop-types';
+
+import Header from '../../components/Header';
+import FacebookButton from '../../components/FacebookButton';
+import GoogleButton from '../../components/GoogleButton';
+import './signIn.css';
+import * as ROUTES from '../../constants/routes';
+
+const SignInForm = props => {
+  const {
+    form: { getFieldDecorator, validateFieldsAndScroll },
+    history: { goBack },
+  } = props;
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    validateFieldsAndScroll((err, values) => {
+      if (!err) {
+        // eslint-disable-next-line no-console
+        console.log('Received values of form: ', values);
+      }
+    });
+  };
+
+  return (
+    <div className="signin">
+      <Header text="Sign In" handleBack={goBack} />
+
+      <section className="signin__form">
+        <Form onSubmit={handleSubmit}>
+          <Form.Item hasFeedback>
+            {getFieldDecorator('email', {
+              rules: [
+                {
+                  type: 'email',
+                  message: 'The input is not valid E-mail!',
+                },
+                {
+                  required: true,
+                  message: 'Please input your E-mail!',
+                },
+              ],
+            })(
+              <Input
+                prefix={<Icon type="mail" className="signin__icon" />}
+                placeholder="Email"
+              />
+            )}
+          </Form.Item>
+
+          <Form.Item hasFeedback>
+            {getFieldDecorator('password', {
+              rules: [
+                {
+                  pattern: new RegExp(
+                    /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d].{8,}$/
```

Comment:
we don't need the regex here ?!! 
in the sign-in we just need it to be required